ledger/participant-integration-api/src/main/scala/platform/store/backend/postgresql/PostgresDbBatch.scala

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.store.backend.postgresql
+
+import com.daml.platform.store.backend.DBDTOV1
+import com.daml.scalautil.NeverEqualsOverride
+
+import scala.reflect.ClassTag
+
+case class PostgresDbBatch(
```

Comment:
Can you make these private to `store`?

Reply:
yeah, good idea, I'll seal off visibility for the entire package

Reply:
actually since the existential type on the factory this can be also private[postgresql]